cli.py

```diff
@@ -112,25 +114,20 @@ def load_client_script(base_setup_path, setup_module_name):
         log.error("Could not find the specified module")
 
 
-def set_simulation_file_env(base_setup_path, simulation_config_path, run_on_redis):
+def create_simulation_config_path(base_setup_path, simulation_config_path, run_on_redis):
     if run_on_redis is True:
-        os.environ["SIMULATION_CONFIG_FILE_PATH"] = ""
-        return
-    if simulation_config_path is None:
-        raise ApiSimulationConfigException(f"simulation_config_path must be provided")
+        return ""
```

Comment:
The reason why I suggested removing `run_on_redis` from this function is that the path to the simulation config can/should be created regardless of Redis. If you are using Redis, then you don't need the simulation config path at all. If you are not using Redis, then you need to compute the path. But the check for Redis doesn't belong to the logic for the costruction of the path.
If you think about it in terms of unit testing and atomicity of tasks, it makes sense to split the two things.

Reply:
https://github.com/gridsingularity/d3a-api-client/pull/94/commits/502901535a99d23ce71dc26480ac8f9c7da5a49e